extensions/typescript-language-features/extension-browser.webpack.config.js

```diff
@@ -64,18 +64,20 @@ module.exports = withBrowserDefaults({
 		new CopyPlugin({
 			patterns: [
 				{
-					from: '../node_modules/typescript/lib/tsserver.js',
+					from: '../node_modules/typescript/lib/tsserverlibrary.js',
 					to: 'typescript/tsserver.web.js',
 					transform: async (content) => {
 						const dynamicImportCompatPath = path.join(__dirname, '..', 'node_modules', 'typescript', 'lib', 'dynamicImportCompat.js');
+						const hostpath = path.join(__dirname, 'out', 'tsServer', 'webServer.js');
```

Comment:
This seems like an okay hack, but I think long-term this probably just needs to use webServer as the entrypoint and bundle it directly. The webServer file would then do:

```
import * as ts from "typescript/lib/tsserverlibrary"
```

Reply:
I agree, but right now I think it's required because of the dynamic import hack. See the last comment.

Reply:
With this approach, can we inline the `dynamicImport` logic? I think it is only used in `importPlugin`, right?

Reply:
Yes, it in now inlined as a normal `import` call.

Reply:
tsserver.web.js now uses webServer as its entrypoint.